### PR TITLE
Ensure SLOW_LOG_FILE env variable is always set

### LINKED ISSUE #295: TiDB server failed to start due to un-set parameter
The log of failing pod:
```sh
/usr/local/bin/tidb_start_script.sh: line 40: SLOW_LOG_FILE: parameter not set
```

### PR DESCRIPTION
**Pull request by aylei**
close #295 

Signed-off-by: Aylei <rayingecho@gmail.com>

### CODE REVIEW

**Comment by weekface** [comment]
I think you should move #297 codes to this PR, or we can't confirm this PR can work as expected without the e2e tests.

**Comment by aylei** [comment]
Branch issue-296 merged, close #297 #296 
&
/run-e2e-tests

**Comment by aylei** [comment]
@weekface @tennix PTAL

File: tests/e2e/test_helper.go [hunk 1]
```diff
@@ -42,36 +43,53 @@ var (
...
 		},
 	},
 	{
-		ns:          "ns-1",
-		clusterName: "cluster-name-2",
+		clusterSpec: clusterSpec{
+			ns:          "ns-1",
+			clusterName: "cluster-name-2",
+			values: map[string]string{
+				"tidb.separateSlowLog": "true",
```
Comment (weekface) [inline tests/e2e/test_helper.go]: Need a case with `"tidb.separateSlowLog": "false"`?

Comment (aylei) [reply tests/e2e/test_helper.go]: This option is default to false, but in case of we change the defaults later, I will explicitly declare that case.

**Comment by weekface** [comment]
I rebuilt the dind env.

You should rerun the e2e after this job is completed: https://internal.pingcap.net/jenkins/job/rebuild_k8s_dind_test_env/82/

**Comment by aylei** [comment]
ping @tennix @xiaojingchen

**Pull request merged**